enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/AuthFilter.java

```diff
@@ -84,7 +85,8 @@ private void initializeAuthenticators(APIConfig apiConfig) {
             Authenticator jwtAuthenticator = new JWTAuthenticator();
             authenticators.add(jwtAuthenticator);
         }
-        Authenticator authenticator = new InternalAPIKeyAuthenticator(APIConstants.JwtTokenConstants.INTERNAL_KEY);
```

Comment:
Shall we remove APIConstants.JwtTokenConstants.INTERNAL_KEY constant if there is no usage now?

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2013/commits/3e427447869175c37913b0a0a08643fc6dda25f8